plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -73,7 +73,10 @@ def __init__(self, repo, publish_conduit, config, distributor_type, association_
         self.rpm_step = PublishRpmStep(dist_step, repo_content_unit_q=association_filters)
         self.add_child(self.rpm_step)
         self.add_child(PublishDrpmStep(dist_step, repo_content_unit_q=association_filters))
-        self.add_child(PublishErrataStep(repo_content_unit_q=association_filters))
+        errata_step = PublishErrataStep()
+        if config.get_boolean(constants.INCREMENTAL_EXPORT_REPOMD_KEYWORD):
+            errata_step = PublishErrataStep(repo_content_unit_q=association_filters)
+        self.add_child(errata_step)
```

Comment:
Just a style thing, but why define it just to re-define it?

```python
        if config.get_boolean(constants.INCREMENTAL_EXPORT_REPOMD_KEYWORD):
            errata_step = PublishErrataStep(repo_content_unit_q=association_filters)
        else:
            errata_step = PublishErrataStep()
        self.add_child(errata_step)
        # or...
        errata_step_kwargs = {}
        if config.get_boolean(constants.INCREMENTAL_EXPORT_REPOMD_KEYWORD):
            errata_step_kwargs['repo_content_unit_q'] = association_filters
        self.add_child(PublishErrataStep(**errata_step_kwargs))
```

Also, a comment explaining why the errata step only needs `repo_content_unit_q` set in certain circumstances might be nice.